api/v1beta1/temporalcluster_types.go

```diff
@@ -743,6 +743,15 @@ type PrometheusSpec struct {
 type MetricsSpec struct {
 	// Enabled defines if the operator should enable metrics exposition on temporal components.
 	Enabled bool `json:"enabled"`
+	// PerUnitHistogramBoundaries defines the default histogram bucket boundaries.
+	// Configuration of histogram boundaries for given metric unit.
+	//
+	// Supported values:
+	// - "dimensionless"
+	// - "milliseconds"
+	// - "bytes"
+	// +optional
+	PerUnitHistogramBoundaries map[string][]string `json:"perUnitHistogramBoundaries,omitempty"`
```

Comment:
```suggestion
	PerUnitHistogramBoundaries map[string][]float64 `json:"perUnitHistogramBoundaries,omitempty"`
```

The original definition is `float64`

```
		PerUnitHistogramBoundaries map[string][]float64 `yaml:"perUnitHistogramBoundaries"`
```

See https://github.com/temporalio/temporal/blob/893a8f3d51567f889dd6e7f973196f7d94bb1cbe/common/metrics/config.go#L75

Reply:
> found float, the usage of which is highly discouraged, as support for them varies across languages. Please consider serializing your float as string instead. If you are really sure you want to use them, re-run with crd:allowDangerousTypes=true
Error: not all generators ran successfully 
run `controller-gen rbac:roleName=manager-role crd webhook paths=./... output:crd:artifacts:config=config/crd/bases -w` to see all available markers, or `controller-gen rbac:roleName=manager-role crd webhook paths=./... output:crd:artifacts:config=config/crd/bases -h` for usage make: *** [manifests] Error 1

It looks like the usage of float is highly discouraged when generating crd.yaml.